core/src/logging/filter/mod.rs

```diff
@@ -443,9 +493,13 @@ impl InnerLogFilter {
         addon_allowed(&self.allowed_global_ids, id, &addon_filter)
             || addon_allowed_in_origin(&self.allowed_modules, id, origin, &addon_filter)
     }
+
+    pub fn builder() -> FilterConfigBuilder {
```

Comment:
adding `log_level` as parameter here would remove the necessity for the `level()` function in the builder.

Reply:
See comment above.